.changeset/pretty-melons-jog.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/adapter-node': patch
+---
+
+Add test for malformed url
```

Comment:
A changelog is not needed since this doesn't affect the "production code"

Reply:
Done